dashboard/config/blocks/GamelabJr/gamelab_hideVariable.json

```diff
@@ -0,0 +1,20 @@
+{
+  "category": "Variables",
+  "config": {
+    "func": "hideVariable",
+    "blockText": "hide variable {VAR}",
+    "color": [
```

Comment:
Nit: Could we remove this color property now (from both blocks)? The preview in the block editor on your branch was probably showing it teal because it was still using CDO Blockly. If you merge in staging, it should load http://localhost-studio.code.org:3000/pools/GamelabJr/blocks/gamelab_hideVariable/edit with Google Blockly and you'll be able to confirm that the style is sufficient.

Reply:
Ah, good catch, thank you!